context/app/static/js/components/detailPage/summary/Summary/Summary.jsx

```diff
@@ -23,10 +18,13 @@ function Summary({
   collectionName,
   mapped_external_group_name,
 }) {
+  const {
```

Comment:
This `Summary` component is also used on a lot of other pages (Collection, Dataset, and Donor), we should double check functionality there as well

Reply:
Checked, all are working!